integrations/jenkins/overrides.py

```diff
@@ -0,0 +1,23 @@
+import typing
+
+from port_ocean.core.handlers.port_app_config.models import (
+    ResourceConfig,
+    PortAppConfig,
+    Selector,
+)
+from pydantic import Field
+
+
+class JenkinStagesResourceConfig(ResourceConfig):
+    class JenkinStageSelector(Selector):
+        query: str
+        job_url: str = Field(alias="jobUrl", required=True)
```

Comment:
in terms of the user experience, I was just wondering why we're using the job_url instead of the job_name. is the job_url easier or the same as the name? maybe you can share a sample format for the job_url

Reply:
In terms of user experience, we use the job_url instead of the job_name **primarily for consistency and reliability**. Job names can contain spaces or special characters, which require URL encoding, and this can lead to issues if not handled correctly. On the other hand, the job_url is a complete and fully encoded reference to the job, making it more straightforward to work with.

Here are some examples:

```
http://localhost:8080/job/Phalbert/job/airframe-react/
http://localhost:8080/job/limbopay/job/Limbo%20Core/job/main/
```

The second url would have the job name` Limbo Core` in the jenkins dashboard. 

In this format, the job_url already includes everything necessary to make API requests (such as appending `/api/json`), eliminating the need for additional encoding steps. By using job_url, we avoid potential errors related to name encoding and ensure the API calls are more predictable.

After all, the Job URL is always returned as a property and a user can copy and paste in the mapping.